src/main.rs

```diff
@@ -30,36 +30,38 @@ Options:
 #[derive(Debug, Deserialize)]
 struct Args {
     flag_pattern: String,
-    arg_folder: String
+    arg_list_id: String,
+    arg_folder: String,
 }
 
 /// Returns true if entry is a file, false otherwise or on error.
 fn is_file(entry: &fs::DirEntry) -> bool {
-    entry.metadata().ok()
-        .map(|m| m.is_file())
-        .unwrap_or(false)
+    entry.metadata().ok().map(|m| m.is_file()).unwrap_or(false)
 }
 
 /// Used in to filter_map entries into files. Directories are sorted out.
 fn into_file(entry: io::Result<fs::DirEntry>) -> Option<fs::DirEntry> {
-    entry.ok().and_then(|e| {
-        if is_file(&e) {
-            Some(e)
-        } else {
-            None
-        }
-    })
+    entry.ok().and_then(
+        |e| if is_file(&e) { Some(e) } else { None },
```

Comment:
I wonder if the comma at the end is a bug.

Reply:
Nope. Just uniformly always added.